src/python/CRABClient/JobType/Analysis.py

```diff
@@ -40,6 +40,7 @@ def run(self, filecacheurl = None):
         Override run() for JobType
         """
         configArguments = {'addoutputfiles'            : [],
+                           'adduserfiles'              : [],
```

Comment:
Hey Stefano, looks like you were using an outdated client branch, this line shouldn't be here, was removed in this commit https://github.com/dmwm/CRABClient/commit/7502a559246128422edf088b57197552f23fc652